<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="BACLARK77-MAC1.local" name="com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS" tests="3" failures="0" timestamp="3 Sep 2014 19:34:53 GMT" time="210.031" errors="3">
  <testcase name="ImplementMPS_TC2901" time="61.394" classname="com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS">
    <error type="java.lang.AssertionError" message="Source &#039;var mpscall = {&quot;site&quot;:&quot;pub7-development&quot;,&quot;title&quot;:&quot;Welcome to Site-Install&quot;,&quot;path&quot;:&quot;\/&quot;,&quot;is_content&quot;:0,&quot;type&quot;:&quot;other&quot;}&#039; is not present in page. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Source 'var mpscall = {"site":"pub7-development","title":"Welcome to Site-Install","path":"\/","is_content":0,"type":"other"}' is not present in page. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.nbcuni.test.publisher.pageobjects.Content.ContentParent.VerifySourceInPage(ContentParent.java:163)
	at com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS.ImplementMPS_TC2901(ImplementMPS.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- ImplementMPS_TC2901 -->
  <testcase name="ImplementMPS_TC2901" time="60.774" classname="com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS">
    <error type="java.lang.AssertionError" message="Source &#039;var mpscall = {&quot;site&quot;:&quot;pub7-development&quot;,&quot;title&quot;:&quot;Welcome to Site-Install&quot;,&quot;path&quot;:&quot;\/&quot;,&quot;is_content&quot;:0,&quot;type&quot;:&quot;other&quot;}&#039; is not present in page. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Source 'var mpscall = {"site":"pub7-development","title":"Welcome to Site-Install","path":"\/","is_content":0,"type":"other"}' is not present in page. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.nbcuni.test.publisher.pageobjects.Content.ContentParent.VerifySourceInPage(ContentParent.java:163)
	at com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS.ImplementMPS_TC2901(ImplementMPS.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- ImplementMPS_TC2901 -->
  <testcase name="ImplementMPS_TC2901" time="87.863" classname="com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS">
    <error type="java.lang.AssertionError" message="Source &#039;var mpscall = {&quot;site&quot;:&quot;pub7-development&quot;,&quot;title&quot;:&quot;Welcome to Site-Install&quot;,&quot;path&quot;:&quot;\/&quot;,&quot;is_content&quot;:0,&quot;type&quot;:&quot;other&quot;}&#039; is not present in page. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Source 'var mpscall = {"site":"pub7-development","title":"Welcome to Site-Install","path":"\/","is_content":0,"type":"other"}' is not present in page. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.nbcuni.test.publisher.pageobjects.Content.ContentParent.VerifySourceInPage(ContentParent.java:163)
	at com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS.ImplementMPS_TC2901(ImplementMPS.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- ImplementMPS_TC2901 -->
=======
<testsuite hostname="BACLARK77-MAC1.local" name="com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS" tests="1" failures="0" timestamp="8 Sep 2014 20:19:08 GMT" time="119.993" errors="0">
  <testcase name="ImplementMPS_TC2901" time="119.993" classname="com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS"/>
>>>>>>> refs/heads/Brandon
</testsuite> <!-- com.nbcuni.test.publisher.tests.Advertising.MPS.ImplementMPS -->
